docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,46 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
+
+**Versioned name**: *Round-5*
+
+**Category**: Arithmetic unary operation 
+
+**Short description**: *Round* performs element-wise round operation with given tensor.
+
+**Detailed description**: Operation takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer.
+    
+    round([-4.5, -1.9, -1.5, 0.9, 1.5, 2.3, 2.5, 1.5]) = [-4.0, -2.0, -2.0, 1.0, 2.0, 2.0, 2.0, 2.0]
```

Comment:
maybe for better demonstration you can move the result to the next line and align with input. How do you think?

Reply:
Done, also added an example for `half_away_from_zero` mode.